app/src/main/res/values/strings.xml

```diff
@@ -91,7 +91,6 @@
     <string name="sort_description">Sort entries</string>
     <string name="search_clear_content_description">Clear search bar text</string>
     <string name="profile_image_content_description">Firefox Account Profile Image</string>
-    <string name="app_version">Application version number</string>
```

Comment:
Is this intentional for this PR?

Reply:
just some cleanup when I was fixing the baseline lint file 🛀